components/dashboard/src/App.tsx

```diff
@@ -72,4 +67,13 @@ const App: FC = () => {
     );
 };
 
-export default AppWithErrorBoundary;
+// Routing level above main App component for any routes that don't need user/orgs loaded, such as addressable error pages
```

Comment:
This adds a layer above our App to let us render things like addressable error pages w/ their own routes. They won't load any external data by default.